lib/experimental/fonts/font-library/class-wp-font-family-utils.php

```diff
@@ -122,4 +122,41 @@ function ( $family ) {
 
 		return $font_family;
 	}
+
+	/**
+	 * Sanitizes the font family data using WP_Theme_JSON.
+	 *
+	 * @since 6.5.0
+	 *
+	 * @param string $data string to sanitize.
+	 * @return array A sanitized font family definition.
+	 */
+	public static function sanitize( $data ) {
+		// Creates the structure of theme.json array with the new fonts.
+		$fonts_json = array(
+			'version'  => '2',
+			'settings' => array(
+				'typography' => array(
+					'fontFamilies' => array(
+						'custom' => array(
+							json_decode( $data, true ),
+						),
+					),
+				),
+			),
+		);
+
+		// Creates a new WP_Theme_JSON object with the new fonts to
+		// leverage sanitization and validation.
+		$fonts_json = WP_Theme_JSON_Gutenberg::remove_insecure_properties( $fonts_json );
+		$theme_json = new WP_Theme_JSON_Gutenberg( $fonts_json );
+		$theme_data = $theme_json->get_data();
+		$sanitized  = ! empty( $theme_data['settings']['typography']['fontFamilies'] )
+			? $theme_data['settings']['typography']['fontFamilies'][0]
+			: array();
+
+		$sanitized['slug']       = _wp_to_kebab_case( $sanitized['slug'] );
```

Comment:
Can it always be guaranteed that `slug` and `fontFamily` will exist? Should we add a guard here?

Reply:
Maybe we should be checking for these anyway, I'm always a fan of putting guards in.

Reply:
Done in https://github.com/WordPress/gutenberg/pull/56172/commits/832db93b962481affb41d21cd8b41ab479634f59.